src/number-formatting/format.js

```diff
@@ -12,6 +12,37 @@ function formatNegativeNumber(str, options) {
 }
 
 /**
+ * converts a number to a decimal string if it is on scientific notation
+ * (method taken from https://gist.github.com/jiggzson/b5f489af9ad931e3d186)
+ * @param number
+ */
+function convertScientificToDecimal(number) {
+    // if the number is in scientific notation remove it
+    if(/\d+\.?\d*e[\+\-]*\d+/i.test(number)) {
+        var zero = '0';
+        var parts = String(number).toLowerCase().split('e');
+        var exponential = parts.pop();
+        var zerosLength = Math.abs(exponential);
+        var sign = exponential / zerosLength;
+        var coefficient = parts[0].split('.');
+
+        if (sign === -1) {
+            number = zero + '.' + new Array(zerosLength).join(zero) + coefficient.join('');
+        } else {
+            var coefficientDecimal = coefficient[1];
+
+            if (coefficientDecimal) {
+				zerosLength = zerosLength - coefficientDecimal.length;
+			}
+
+            number = coefficient.join('') + new Array(zerosLength + 1).join(zero);
```

Comment:
I'm afraid not, unless that `lodash` was being used, where a `_.repeat` helper method exists.